app.js

```diff
@@ -1,11 +1,16 @@
 const persist = require('choo-persist')
 const mount = require('choo/mount')
 const log = require('choo-log')
+const css = require('sheetify')
```

Comment:
@Kriesse I like the words `css` and `html` for `sheetify` and `bel` respectively because github detects these and applies syntax highlighting. The rest is usually camelcased stuff (: